civis/cli/__main__.py

```diff
@@ -130,11 +136,12 @@ def param_case_map(param_names):
     return result
 
 
-def invoke(method, path, op, *args, json_output=False, **kwargs):
+def invoke(method, path, op, *args, **kwargs):
```

Comment:
Done, though I don't know how useful it is without any other accompanying documentation.